### PR TITLE
 uploadind Assignment_09

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

**Review by Priyankashelke2411 — APPROVED** [review]

File: Nikita/Assignment_9/Ass_09.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+left: 922px;
+top: 275px;
+}
+h1{
+position: absolute;
+margin-top: 50px;
+width: 396px;
+height: 125px;
+bottom: 100px;
+}
```
Comment (Pawan1005) [inline Nikita/Assignment_9/Ass_09.css]: add new line at the end of files

File: Nikita/Assignment_9/Ass_09.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="jobs-list2">
+        <ul>
+                <li><a href="#"><img src="d.png" alt="2"></li>
+                <li><a href="#"><img src="e.png" alt="2"></li>
+                <li><a href="#"><img src="f.png" alt="2"></li>
+        </ul>
+    </div>
+    <div class="state"><img src="g.png" alt="g"></div>
+</body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Assignment_9/Ass_09.html:49]: indentation is wrong make it correct

File: Nikita/Assignment_9/Ass_09.css (hunk 1 above)
Comment (sourabh513) [inline Nikita/Assignment_9/Ass_09.css:96]: remove extra line

Comment (sourabh513) [inline Nikita/Assignment_9/Ass_09.css]: add extra new line at the end of programe

File: Nikita/Assignment_9/Ass_09.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            </span>
+        </form>
+    </div>
+    <div class="job"><img src="find.png" alt="find"></div>
+    <div class="item"><img src="6.png" alt="6"></div>
+    <div class="display"><img src="7.png" alt="7"></div>
+    <h1>Browse Job</h1>
+    <div class="jobs-list1">
+        <ul>
+                <li><a href="#"><img src="a.png" alt="2"></li>
```
Comment (sourabh513) [inline Nikita/Assignment_9/Ass_09.html]: indentation wrong

**Review by sourabh513 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by sourabh513 — APPROVED** [review]

File: Nikita/Assignment_9/Ass_09.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Ass_09.css">
+    <title>Assignment_09</title>
+</head>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_9/Ass_09.html:9]: check indentation after html tag

File: Nikita/Assignment_9/Ass_09.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="inner">
+        <form>
+            <input type="text" id="fname"  placeholder="Enter skill, Designation etc..">
+            <span class="inpu">
+            <input type="text" id="name"  placeholder="Enter location">
+            <input type="text" id="jes"  placeholder="Experience">
+            </span>
+        </form>
+    </div>
+    <div class="job"><img src="find.png" alt="find"></div>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_9/Ass_09.html:29]: check indentation

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**